app/views/shared/_flash.html.erb

```diff
@@ -1,7 +1,8 @@
 <%= turbo_frame_tag "flash" do %>
   <% flash.each do |key, message| %>
-    <div class="<%= flash_class(key) %> alert-dismissible" role="alert">
-      <%= message %>
-    </div>
+    <div
+      class="<%= flash_class(key) %> alert-dismissible"
+      role="alert"
+      style="white-space: pre-line;"><%= message %></div>
```

Comment:
`errors.join("<br>".html_safe)` didn't work. It just renders at `stuff<br>stuff` instead of working like a real html element. Open to suggestions but this styling change makes the `\n` be respected.